modules/ROOT/pages/configuration/configuration-settings.adoc

```diff
@@ -3901,6 +3900,22 @@ m|
 |===
 
 
+[role=label--new-5.20]
+[[config_dbms.security.http_static_content_security_policy_header]]
+=== `dbms.security.http_static_content_security_policy_header`
+
+.dbms.security.http_static_content_security_policy_header
+[frame="topbot", stripes=odd, grid="cols", cols="<1s,<4"]
+|===
+|Description
+a|Defines the Content-Security-Policy header to return to content returned on static endpoints.
+|Valid values
+a|A string.
+|Default value
+m|+++default-src 'self'; script-src 'self' cdn.segment.com canny.io; img-src 'self'; style-src 'self' fonts.googleapis.com 'unsafe-inline'; font-src 'self' fonts.gstatic.com; base-uri 'none'; object-src 'none'; frame-ancestors 'none'; connect-src 'self' api.canny.io api.segment.io wss+++
```

Comment:
we just [merged an update](https://github.com/neo-technology/neo4j/pull/25301/files) for the default value:
```suggestion
m|+++default-src 'self'; script-src 'self' cdn.segment.com canny.io; img-src 'self' data:; style-src 'self' fonts.googleapis.com 'unsafe-inline'; font-src 'self' fonts.gstatic.com; base-uri 'none'; object-src 'none'; frame-ancestors 'none'; connect-src 'self' api.canny.io api.segment.io ws: wss: http: https:+++
```